tensorflow/tensorboard/dist/tf-tensorboard.html

```diff
@@ -3812,6 +3830,17 @@ <h3 id="tooltip-help" class="tooltip-container">
           chartScaffold.chart().redraw();
         }
       },
+
+      toggleYScale: function(e) {
+        var b = this.parentNode.querySelector('.log-option-button');
+        var c = this.parentNode.querySelector('vz-line-chart');
+        if (c !== null) {
```

Comment:
As currently written, clicking any log/linear button always modifies the first chart. Here is a fix:
```
      toggleYScale: function(e) {
        var currentTarget = Polymer.dom(e.currentTarget);

        var b = currentTarget.parentNode.querySelector('.log-option-button');
        var c = currentTarget.parentNode.querySelector('vz-line-chart');
        if (c !== null) {
          c.yScale = c.yScale === 'log' ? 'linear' : 'log';
          b.icon = c.yScale === 'log' ? 'line-weight' : 'reorder';
          c.redraw();
        }
      },
```